docker-compose.yml

```diff
@@ -86,6 +86,29 @@ services:
       - --network-endpoint=https://polygon-rpc.com         # network endpoint, can choose your own endpoint
       - --token-duration=24                                # query auth token validity [hours]
       - --redis-endpoint=redis://indexer_cache
+      - --metrics-token=thisismyAuthtoken
+
+  # Uncomment to enable dashboard
+
+  # prometheus:
```

Comment:
Move `prometheus` and `grafana` to a separate file: `dock-compose-metrics.yml` which have the same network with indexer-services